### DISCUSSION TITLE
How to create action directly with zigbee2mqtt frontend

### DISCUSSION THREAD

**Discussion by eonethelama** [Koenkk/zigbee2mqtt#12654] [Devices]
Hello,

I've installed zigbee2mqtt on a raspberry Pi4B by using Rasbian.
I activated the frontend and paired several ikea devices by using the frontend.
I can drive the devices, for example switch on/off a bulb or a A/C outlet, by using the frontend. I also have a simple switch on/off.
Now what I do not understand is how to create automatic action between the devices.

For exemple switch on/off the light of the bulb by using the on/off switch ?
Is it possible by using directly zigbee2mqtt and the frontend ?

I prefer for the moment not using additionnal layer like HomeAssistant or Domoticz.

Thanks
--eone

**Comment by davew21** [comment]
zigbee2mqtt is used to get device information, status and control of zigbee devices into an mqtt broker, the frontend is only for managing the devices, you still need something else to interact with the data passed to the mqtt broker. There are plenty of free home automation platforms out there that will use and manage the informtion and control the devices with timers etc. Personally I use Domoticz but do your research and pick something you think is suitable for you, most of the platforms have good community support.
